ios/FluentUI/Navigation/SideBar.swift

```diff
@@ -0,0 +1,284 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFSideBarDelegate)
+public protocol SideBarDelegate {
+	/// Called after the view representing `TabBarItem` is selected.
+	@objc optional func sideBar(_ sideBar: SideBar, didSelect item: TabBarItem, fromTop: Bool)
+
+	/// Called after the avatar view is tapped in the side bar.
+	@objc optional func sideBar(_ sideBar: SideBar, didActivate avatarView: AvatarView)
+}
+
+@objc(MSFSideBar)
+open class SideBar: UIView {
+	private enum Section: Int {
+		case top
+		case bottom
+	}
+
+	private struct Constants {
+		static let maxTabCount: Int = 5
+		static let viewWidth: CGFloat = 62.0
+		static let avatarViewSize: CGFloat = 30.0
+		static let topItemSpacing: CGFloat = 38.0
+		static let bottomItemSpacing: CGFloat = 28.0
+	}
+
+	@objc public weak var delegate: SideBarDelegate?
+
+	private var avatarViewGestureRecognizer: UITapGestureRecognizer?
+	@objc open var avatarView: AvatarView? {
```

Comment:
Per other comment - I'm initially leaning towards this just having a dependency on a view and not a specific AvatarView, but curious to hear your thoughts

Reply:
See my response to your other comment. I want this to explicitly be AvatarView so we can control the experience.